subgraphs/uniswap-v2/src/core.ts

```diff
@@ -0,0 +1,527 @@
+/* eslint-disable prefer-const */
+import { BigInt, BigDecimal, store, log } from '@graphprotocol/graph-ts'
+import {
+  Deposit,
+  Withdraw, 
+  Swap as SwapEvent,
+  LiquidityPool,
+  DexAmmProtocol,
+  Token,
+  _Transaction, 
+  _Mint as MintEvent, 
+  _Burn as BurnEvent,
+  _HelperStore,
+  _TokenTracker
+} from './../generated/schema'
+
+import { Mint, Burn, Swap, Transfer, Sync } from '../generated/templates/Pair/Pair'
+import { getEthPriceInUSD, findEthPerToken, getTrackedVolumeUSD } from './common/price'
+import {
+  updateFinancials,
+  updateUsageMetrics,
+  convertTokenToDecimal,
+  updatePoolMetrics,
+  updateVolumeAndFees
+} from './common/helpers'
+import {
+  DEFAULT_DECIMALS,
+  FACTORY_ADDRESS,
+  ZERO_ADDRESS,
+  BIGDECIMAL_ZERO,
+  BIGINT_ZERO,
+  BIGDECIMAL_ONE,
+  INT_ONE,
+} from './common/constants'
+
+// Checks if the mint has been completed with the Deposit event. 
+function isCompleteMint(mintId: string): boolean {
+  let mint = MintEvent.load(mintId)
+  if (mint == null) return false
+  return mint.from !== null // sufficient checks
+}
+
+export function handleTransfer(event: Transfer): void {
+
+  // get pair and load contract
+  let pool = LiquidityPool.load(event.address.toHexString())
+
+  if (pool == null) {
+    log.warning('pool was null', [])
+    return
+  }
+
+  let transactionHash = event.transaction.hash.toHexString()
+
+  // user stats
+  let from = event.params.from
+  let to = event.params.to
+
+  // liquidity token amount being transfered
+  let value = convertTokenToDecimal(event.params.value, DEFAULT_DECIMALS)
+
+  // ignore initial transfers for first adds
+  if (to.toHexString() == ZERO_ADDRESS && event.params.value.equals(BigInt.fromI32(1000))) {
```

Comment:
Either change `1000` into a named constant or add a comment explaining what it is

Reply:
Changed to a constant